clientv3/client.go

```diff
@@ -503,3 +503,16 @@ func toErr(ctx context.Context, err error) error {
 	}
 	return err
 }
+
+func canceledByCaller(ctx context.Context, err error) bool {
+	if ctx.Err() == nil {
+		return false
+	}
+	if err == nil {
+		return false
+	}
+	if err != context.Canceled && err != context.DeadlineExceeded {
```

Comment:
Fixed. I just wanted to return `true` at the end of the function (now it's `false`, `false`, `true`, `false`).